dask_cudf/io/tests/test_json.py

```diff
@@ -0,0 +1,47 @@
+import dask
+import dask_cudf
+import dask.dataframe as dd
+from dask.utils import tmpfile
+import pandas as pd
+
+import pytest
+
+
+def test_read_json(tmp_path):
+    df1 = dask.datasets.timeseries(
+        dtypes={"x": int, "y": int}, freq="120s").reset_index(drop=True)
+    df1.to_json(tmp_path / "data-*.json")
+    df2 = dask_cudf.read_json(tmp_path / "data-*.json")
+    dd.assert_eq(df1, df2)
+
+
+@pytest.mark.filterwarnings("ignore:Using CPU")
+@pytest.mark.parametrize('orient', ['split', 'index', 'columns', 'values'])
+def test_read_json_basic(orient):
+    df = pd.DataFrame({'x': ['a', 'b', 'c', 'd'], 'y': [1, 2, 3, 4]})
+    ddf = dd.from_pandas(df, npartitions=2)
+    with tmpfile('json') as f:
+        df.to_json(f, orient=orient, lines=False)
+        actual = dask_cudf.read_json(f, orient=orient, lines=False)
+        actual_pd = pd.read_json(f, orient=orient, lines=False)
+
+        out = actual.compute()
```

Comment:
I don't think you need to call `compute`.  You should be able to pass the ddf object to `assert_eq`